doc/ref/clients/index.rst

```diff
@@ -88,6 +88,12 @@ Salt Caller
 .. autoclass:: salt.client.Caller
     :members: cmd
 
+Salt Proxy Caller
+-----------
```

Comment:
The dashed line needs to match the length of the line above, or we'll get an RST warning.